fastlane/Fastfile

```diff
@@ -145,7 +146,20 @@ def distribute_builds(
       suffix: 'full',
       extension: 'nupkg',
       name: 'Windows Update'
-    }
+    },
+    windows_appx: {
+      binary_path: File.join(BUILDS_FOLDER, 'Studio-appx-signed', "Studio #{normalized_version}.0.appx"),
+      filename_core: 'win32',
+      extension: 'appx',
+      name: 'Windows (Appx)'
+    },
+    windows_appx_unsigned: {
+      binary_path: File.join(BUILDS_FOLDER, 'Studio-appx-unsigned', "Studio #{normalized_version}.0 unsigned.appx"),
+      filename_core: 'win32',
+      suffix: 'unsigned',
+      extension: 'appx',
+      name: 'Windows (Unsigned Appx)'
+    },
```

Comment:
I haven't tried it, but I assume that downloading and opening the unsigned build will result in Windows Defender flagging it as dangerous and possibly refusing to run it.

With that in mind, is it still valuable to add the file to the distribution manifest? 

For what is worth, I think having it just for completeness is a valid answer. At the same time, if it can't run out of the box (without jumping through hoops) I wonder whether we might save the time and storage to upload it.

Reply:
Space is not our concern. If this simplifies the process (which I agree it does 👍) then we're all good.